Feature 7

brtaylor-dev opened the pull request.

brtaylor-dev committed:
feature-7: Finished Chess.java work and am working on Model.java

brtaylor-dev committed:
Merge branch 'master' into feature-7

brtaylor-dev committed:
feature-7: Finished setting up the controller. We now have a program that runs and interprets user commands!

brtaylor-dev committed:
feature-7: Finished up everything else on the trello card. Wrote comments but not test cases. Pulled formatting logic out of model and am leaving the interpretation of validNewPositions to the view to interpret.

brtaylor-dev committed:
Merge branch 'master' into feature-7

brtaylor-dev committed:
feature-7: Added the logging jars to the project

brtaylor-dev committed:
Merge branch 'master' into feature-7

brtaylor-dev committed:
merged 'master' onto 'feature-7' and corrected a few mismatching try/catch statements. Also fixed one comment that IntelliJ was complaining about.

spkaplan commented:
- `Chess.java`
  - [x] End of main, `controller.run()` indent one more
    -`TextController.java`
    - [x]  change `buildcharToIntMap` to `buildCharToIntMap`
    - [x] In "move" case, don't assume two arguments were given
    - [x] In "validMoves" case, don't assume one argument is given
    - [x] Decide how to react to invalid input
- [x] Remove TextControllerTest.java from the commit since it's empty

brtaylor-dev committed:
feature-7: Fixed the following issues:
Chess.java
            End of main, controller.run() indent one more -TextController.java
            change buildcharToIntMap to buildCharToIntMap
            In "move" case, don't assume two arguments were given
            In "validMoves" case, don't assume one argument is given
            Decide how to react to invalid input
 Remove TextControllerTest.java from the commit since it's empty

brtaylor-dev committed:
feature-7: Made a few minor changes, check the logs.

brtaylor-dev committed:
feature-7: Updated the java doc on processInput()
